src/sortByProps.js

```diff
@@ -0,0 +1,37 @@
+import { reduce, either, curry, prop, sort, comparator, map, lt } from 'ramda';
+
+import isNotArray from './isNotArray';
+
+/**
+ * Sort an list of objects by a list of props (if first prop value is equivalent, sort by second, etc).
+ *
+ * @func sortByProps
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.26.0|v2.26.0}
+ * @category List
+ * @sig [{k: v}] -> [String] -> [{k: v}]
```

Comment:
You signature looks like the function consumes data first, which is not the case. More proper signature would be:
```suggestion
 * @sig [k] -> [{k: v}] -> [{k: v}]
```